<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget</class>
 <widget class="QWidget" name="Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>808</width>
    <height>229</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>ServerAPI test</string>
  </property>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>10</y>
     <width>125</width>
     <height>149</height>
    </rect>
   </property>
   <property name="title">
    <string>ConnectState</string>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QRadioButton" name="rbDisconnected">
      <property name="text">
       <string>Disconnected</string>
      </property>
      <property name="checked">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QRadioButton" name="rbConnecting">
      <property name="text">
       <string>Connecting</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QRadioButton" name="rbConnected">
      <property name="text">
       <string>Connected</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QRadioButton" name="rbDisconnecting">
      <property name="text">
       <string>Disconnecting</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QCheckBox" name="cbIsOnline">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>170</y>
     <width>77</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>IsOnline</string>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="btnMyIp">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>150</y>
     <width>131</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>MyIP request</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnCheckUpdate">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>180</y>
     <width>131</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>CheckUpdate request</string>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox_2">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>10</y>
     <width>471</width>
     <height>131</height>
    </rect>
   </property>
   <property name="title">
    <string>ApiResourcesManager</string>
   </property>
   <widget class="QPushButton" name="btnLogin">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>191</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>Login</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnFetchWithAuthHash">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>60</y>
      <width>191</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>FetchWithAuthHash</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnFetchSession">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>90</y>
      <width>191</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>FetchSession</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="edUsername">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>30</y>
      <width>113</width>
      <height>24</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="edPassword">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>30</y>
      <width>113</width>
      <height>24</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="btnRefetchServerCredentials">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>90</y>
      <width>191</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>RefetchServerCredentials</string>
    </property>
   </widget>
  </widget>
  <widget class="QPushButton" name="btnResetFailover">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>150</y>
     <width>101</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>ResetFailover</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
